user/enterprise/worker-configuration.md

```diff
@@ -163,6 +163,65 @@ export TRAVIS_WORKER_DOCKER_BINDS="/tmp:/tmp:rw /var/log"
 A full list of options and mount modes is listed in the official
  [Docker documentation](https://docs.docker.com/storage/bind-mounts/).
 
+## Worker behind a HTTP(S) Proxy
+
+If you're using Travis CI Enterprise behind a HTTP(S) proxy, we've got you covered. Since travis-worker 4.6 it is possible to run builds behind a proxy.
```

Comment:
an HTTP proxy.

Reply:
Ahh, very good point. Let me change that right away.